src/twilio/twilio.service.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable } from '@nestjs/common'
+import { ConfigService } from '@nestjs/config'
+import * as twilio from 'twilio'
+
+@Injectable()
+export class TwilioService {
+  private twilioClient: twilio.Twilio
+  private twilioPhone: string
+  private tempPhone: string
+  private twiml: twilio.TwimlInterface
+  constructor(private readonly configService: ConfigService) {
+    const accountID = this.configService.get<string>('TWILIO_ACCOUNT_ID')
+    const authToken = this.configService.get<string>('TWILIO_AUTH_TOKEN')
+    this.twilioPhone = this.configService.get<string>('TWILIO_PHONE')
+    this.tempPhone = this.configService.get<string>('TEMP_PHONE')
+    this.twilioClient = twilio(accountID, authToken)
+    this.twiml = twilio.twiml
+  }
+
+  async respondMessage(userResponse: string) {
```

Comment:
suggestion: maybe we could rename this and add a comment above the function to explain why this is needed vs. just using `sendMessage`.

```suggestion
  // Used to respond to texts we receive from users
  async respondToMessage(userResponse: string) {
```